### PR TITLE
Problem: missing grpc integration test (fix #194)

### LINKED ISSUE #194: Problem: missing grpc integration test
add grpc integrataion test
1. make tx
2. sign tx
3. broadcast with grpc

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: add query, tx broadcast integration test using grpc

### CODE REVIEW

File: pystarport/pystarport/convert.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (tomtau) [inline pystarport/pystarport/convert.sh:1]: could this be executed on the fly?

**Review by tomtau — COMMENTED** [review]
looks ok, but before merging this, probably good to consider whether to: 1) commit these 50,000 LOC of Python + 180+ files to the repo, or 2) generate this protobuf boilerplate on the fly before test execution (if that's possible)? @yihuang @calvinaco @devashishdxt @linfeng-crypto

File: pystarport/pystarport/convert.sh (hunk 1 above)
Comment (leejw51crypto) [reply pystarport/pystarport/convert.sh:1]: protoc can be installed in nix
grpc plugin should be compiled,
i'll check  grpc plugin in nix

**Comment by leejw51crypto** [comment]
submit again

**Pull request closed without merging**